eve/workers/unit_and_feature_tests/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM spotify/kafka
```

Comment:
why spotify kafka instead of the one we're now using in zenko?

Reply:
Good point, when this was originally made, we didn't have those but I'll test it out with that.

Reply:
I remember having this conversation with Rayenne at that time. His goal was to port whatever we were doing in circle.yml at that time and then do the switch to our kafka later. I think setting it up with our kafka would require pulling in Zookeeper alpine, getting the quorum work, setting up topics etc. I recommend doing that as a different PR.